features/step_definitions/funding_sources_steps.rb

```diff
@@ -86,12 +85,14 @@
 end
 
 # Funding Values
-Given(/^I enter my funding values for grant in aid$/) do
-  @app.funding_values_page.gia_current_year.set "1000"
-  @app.funding_values_page.gia_2015_2016.set "1000"
-  @app.funding_values_page.gia_2016_2017.set "1000"
+# rubocop:disable Metrics/LineLength
+Given(/^I enter my funding values for grant in aid current year "([^"]*)", year2015-2016 "([^"]*)", year2016-2017 "([^"]*)"$/) do |yearone, yeartwo, yearthree|
```

Comment:
Wondering if it's worth splitting this up so that the years are specified in the feature separately?
```
And I enter my funding value for "grant in aid" in "2015-2016" as "1000"
And I enter my funding value for "grant in aid" in "2016-2017" as "1500"
And I enter my funding value for "local levy" in "2015-2016" as "1000"
And I enter my funding value for "local levy" in "2016-2017" as "1500"
```

As it stands, it feels like this step is doing too much

Reply:
This has been updated in the pr feature/funding_sources

Reply:
You mean #80?

Reply:
The Outcome Measures tests never worked so removed - now that I have a range of test steps that can take data from the feature file I will work on Outcome Measure tests in a future pr